docs/install.md

```diff
@@ -184,34 +195,33 @@ pip install -v -e .  # or "python setup.py develop"
 ## A from-scratch setup script
 
 ```shell
+
+# Create conda environment
 conda create -n open-mmlab python=3.8 -y
 conda activate open-mmlab
 
+# Install PyTorch
```

Comment:
Insert `conda install ffmpeg` after env activated.

Reply:
Addressed in https://github.com/open-mmlab/mmhuman3d/pull/237/commits/317ecc4ec07cd447693255294d09cabe8ccbe2bc.